stage0/gc.go

```diff
@@ -55,3 +60,68 @@ func GC(pdir string, uuid *types.UUID, stage1Path string, debug bool) error {
 	}
 	return c.Run()
 }
+
+type mount struct {
+	id         int
+	parentId   int
+	mountPoint string
+}
+
+type mounts []*mount
+
+// Sort in an order we can unmount; child before parent.
+func (m mounts) Less(i, j int) bool { return m[i].id != m[j].parentId }
```

Comment:
I guess you already changed this code, right? (Not sure if doing `!=` in `Less` function is the right thing to do).